packages/preview/touying-pres-ustc/0.2.0/content/example/main.typ

```diff
@@ -0,0 +1,403 @@
+#import "@preview/touying-pres:0.2.0": *
```

Comment:
```suggestion
#import "@preview/touying-pres-ustc:0.2.0": *
```

Reply:
Oops, by coincidence, I had a local copy of this package, so I didn’t realize the mistake. I have corrected it now. Thank you!